src/ResourceManager/DeviceProvisioningServices/AzureRM.DeviceProvisioningServices.psd1

```diff
@@ -0,0 +1,134 @@
+FunctionsToExport = @()
+
+# Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
+CmdletsToExport = 'New-AzureRmIoTDeviceProvisioningService', 'Get-AzureRmIoTDeviceProvisioningService',
+				  'Remove-AzureRmIoTDeviceProvisioningService', 'Update-AzureRmIoTDeviceProvisioningService',
+				  'Get-AzureRmIoTDeviceProvisioningServiceAccessPolicy', 'Add-AzureRmIoTDeviceProvisioningServiceAccessPolicy',
+				  'Remove-AzureRmIoTDeviceProvisioningServiceAccessPolicy', 'Update-AzureRmIoTDeviceProvisioningServiceAccessPolicy',
+				  'Get-AzureRmIoTDeviceProvisioningServiceLinkedHub', 'Add-AzureRmIoTDeviceProvisioningServiceLinkedHub',
+				  'Remove-AzureRmIoTDeviceProvisioningServiceLinkedHub', 'Update-AzureRmIoTDeviceProvisioningServiceLinkedHub',
+				  'Get-AzureRmIoTDeviceProvisioningServiceCertificate', 'Add-AzureRmIoTDeviceProvisioningServiceCertificate',
+				  'Remove-AzureRmIoTDeviceProvisioningServiceCertificate', 'Set-AzureRmIoTDeviceProvisioningServiceCertificate',
+				  'Get-AzureRmIoTDeviceProvisioningServiceCertificateVerificationCode'
+
+# Variables to export from this module
+# VariablesToExport = @()
+
+# Aliases to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no aliases to export.
+AliasesToExport = 'New-AzureRmIoTDps', 'Get-AzureRmIoTDps', 'Remove-AzureRmIoTDps', 'Update-AzureRmIoTDps',
+				  'Get-AzureRmIoTDpsAccessPolicy', 'Add-AzureRmIoTDpsAccessPolicy', 'Remove-AzureRmIoTDpsAccessPolicy',
+				  'Update-AzureRmIoTDpsAccessPolicy', 'Get-AzureRmIoTDpsHub', 'Add-AzureRmIoTDpsHub',
+				  'Remove-AzureRmIoTDpsHub', 'Update-AzureRmIoTDpsHub', 'Get-AzureRmIoTDpsCertificate', 'Add-AzureRmIoTDpsCertificate',
+				  'Remove-AzureRmIoTDpsCertificate', 'Set-AzureRmIoTDpsCertificate', 'Get-AzureRmIoTDpsCVC'
+
+# DSC resources to export from this module
+# DscResourcesToExport = @()
+
+# List of all modules packaged with this module
+# ModuleList = @()
+
+# List of all files packaged with this module
+# FileList = @()
+
+# Private data to pass to the module specified in RootModule/ModuleToProcess. This may also contain a PSData hashtable with additional module metadata used by PowerShell.
+PrivateData = @{
+
+    PSData = @{
+
+        # Tags applied to this module. These help with module discovery in online galleries.
+        Tags = 'Azure','ResourceManager','ARM','DPS','IoTHub Device Provisioning Service'
+
+        # A URL to the license for this module.
+        LicenseUri = 'https://aka.ms/azps-license'
+
+        # A URL to the main website for this project.
+        ProjectUri = 'https://github.com/Azure/azure-powershell'
+
+        # A URL to an icon representing this module.
+        # IconUri = ''
+
+        # ReleaseNotes of this module
+        # ReleaseNotes = ''
+
```

Comment:
@anusapan if you updated your version of `PowerShellGet` to the 1.6.0 or later (the latest), when you do `Update-ModuleManifest` on this `.psd1` file, it will include a section for `Prelease`, which you should set to `preview`

Reply:
Done